### ISSUE TITLE
Since the latest update, Atom won't launch

### ISSUE THREAD

**Issue by monetteContensive** [atom/atom#18538]
I have been using Atom for almost 3 years now, but with the recent update, since last week, Atom won't even launch.  I tried uninstalling it and reinstalling it to no avail, so then I tried the Beta version; still no luck.

Not sure what is causing this, but in the meantime, I'm using other editors which I don't like nearly as much as Atom :-( 
I'm on a Windows 10 platform.

**Comment by rsese** [comment]
Thanks for reaching out!

As asked in the issue template, have you tried [safe mode](https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode) or any of the other suggestions in the [debugging guide](https://flight-manual.atom.io/hacking-atom/sections/debugging/)?

If you still have trouble after trying those suggestions, please fill out the issue template details and we can re-open to take a look.

---

We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

Thanks for understanding and meeting us half way :grinning:

**Comment by monetteContensive** [comment]
I did try starting it in safe mode... although I can see Atom as being
installed in my programs under apps and features, when I try initiating
from the command prompt, it says, 'atom' is not recognized as an internal
or external command......
It also does not appear in the programs under the windows startup...
essentially, it's there, but not there.  What's weird is atom has been
working fine on this laptop for two years now.  It only stopped working
about two weeks ago.   Again, I uninstalled and try to reinstall and this
only seemed to make it worse.  Not sure what is going on here.
